src/components/calendar/calendar.e2e.ts

```diff
@@ -6,8 +6,9 @@ import { waitForCondition, waitForLitRender, EventSpy } from '../core/testing';
 
 import { SbbCalendarElement } from './calendar';
 
-describe('sbb-calendar', () => {
-  const selected = new Date(2023, 0, 15).getTime() / 1000;
+import '../button';
+
+describe(`sbb-calendar with ${fixture.name}`, () => {
```

Comment:
probably doesn't matter, but this change will come with your next pr :)
```suggestion
describe(`sbb-calendar`, () => {
```